### ISSUE TITLE
estimating head pose on several face

### ISSUE THREAD

**Issue by GhTara** [WIKI2020/FacePose_pytorch#47]
thanks so. I have an question, is this model acceptable on several face in a frame to estimate head pose?

**Comment by WIKI2020** [comment]
Of course, you can try to modify the code yourself, which may be a little complicated

**Comment by GhTara** [comment]
> Of course, you can try to modify the code yourself, which may be a little complicated

very good, thanks.
